### DISCUSSION TITLE
Bad close issue status color

### DISCUSSION THREAD

**Discussion by zxh326** [community/community#6801] [Projects and Issues]
I often thought it was the pr that had been merged...

**Comment by timoyates** [comment]
Hi @zxh326, thanks for your feedback! We're continuing to iterate on our new Issue color and states, so stay tuned. Cheers!

**Reply by krystian3w** [reply]
Just stop and generate questionnaire first.

**Comment by Katsute** [comment]
I've created a quick chrome extension if you want to revert it back to red: https://github.com/Katsute/GitHub-Red-Closed-Issues-Extension

**Reply by zxh326** [reply]
thanks!

**Reply by krystian3w** [reply]
Add legacy  view for releases too?

**Reply by Katsute** [reply]
@krystian3w
Adding legacy releases is outside the scope of that particular project and more difficult for me to implement. You should find another developer to implement this or send feedback to GitHub on which particular features from the old releases you want in the new releases.

**Reply by iam-py-test** [reply]
@Katsute does your extension support Firefox?
I see Chrom(ium) but not Firefox.

**Reply by iam-py-test** [reply]
<details>
<summary>Not really</summary>
Not really:
<img src="https://user-images.githubusercontent.com/84232764/139533516-f2fc0ff3-5b27-4b70-9072-da365d7f3347.png">
I think it is because Mozilla has not implemented their version of Manifest V3 
Changing the manifest version makes it work again
<img src="https://user-images.githubusercontent.com/84232764/139533668-490de321-c6e6-4f74-9893-9dd4691213b5.png">
</details>

**Reply by krystian3w** [reply]
why not simple user.css or user.js?

**Reply by iam-py-test** [reply]
> why not simple user.css or user.js?

Not sure how to use those. Besides, it only requires a simple change to make it work in both browsers

**Reply by TheBrokenRail** [reply]
> @Katsute does your extension support Firefox? I see Chrom(ium) but not Firefox.

I looked into porting it myself. The actual extension works fine in Firefox but you need to downgrade the manifest version. The real issue is that Firefox doesn't support installing unsigned extensions. So you would have to send the extension through Mozilla's signing process to have it installed permanently. Either that or use a version of Firefox that supports disabling signature verification.

**Reply by Katsute** [reply]
I've downgraded the manifest version from 3 to 2 in the latest version 👍

**Reply by iam-py-test** [reply]
> 
> 
> I've downgraded the manifest version from 3 to 2 in the latest version 👍

I guess I will delete my fork. Thanks!

**Reply by TheBrokenRail** [reply]
You should submit it to Firefox's addons page, it looks pretty painless: https://extensionworkshop.com/documentation/publish/submitting-an-add-on,

**Comment by TheBrokenRail** [comment]
I agree that the new closed issue color needs to be reverted back to red.

Pull requests have 3 main states, represented by 3 main colors: open is green, closed is red, and merged is purple. And as such, it is quite easy to start associating these colors with their states, it makes it easy to tell at a glance what state something is in.

And this carries over to issues, except the merge state is dropped. Or at least it *did* carry over to issues. Now that purple represents both merged and closed depending on the situation, that mental heuristic is worse than useless.

And that is why I believe the change should be reverted.

**Comment by atom0s** [comment]
This just feels like another rushed design choice with no actual purpose. Change for the sake of change is not good. Please stop doing it.

As others have stated, this should have been a beta'd feature with your preview system. Because this is something that would have been immediately turned off/unwanted by everyone who tested it.

Also, if you open your statements with things like:

> Our current issue icon colors are a source of constant user feedback, citing confusion with errors, confusion between why Open is green and Closed is red, accessibility concerns and the general scariness of seeing red across the issues index page when a bunch of closed issues is usually a good thing.

Please show us the proof and numbers to back this up. I have never once seen someone complain about the colors of these icons/buttons; ever. The only time I've ever seen anyone mention color changes on GitHub were for suggestions to those who have color blindness. _(Which is a totally understandable suggestion for change, but as something to opt-into if you need it.)_

These new colors just make it seem like the entire page is accepted and merged pull requests. 
This is not useful or better at all.

**Reply by HeCorr** [reply]
This.

When I started using GitHub I did find the green/red colors a bit confusing but I quickly got used to it.
Though, even it if was confusing, at least it **has always been confusing**. Changing it suddenly would just create even more confusion to the users that are already used to green/red.

**Comment by matievisthekat** [comment]
I agree that red makes more sense to represent a closed issue. On the other hand, adding the purple colour creates an opportunity for a third issue status. For example: open, resolved, closed. Or something along those lines. I think another issue status could potentially be useful, but I'd rather wait for more info before making my judgement

**Reply by JannesPeters** [reply]
that is indeed the plan we have for issue states... you can read more about it here: https://github.com/github/roadmap/issues/289

**Comment by Katsute** [comment]
A Chrome/Firefox extension is available for anyone who wants to revert this change https://github.com/Katsute/GitHub-Red-Issues#readme
